mwdb/web/src/components/RecentView/RecentView.js

```diff
@@ -140,17 +175,53 @@ export default function RecentView(props) {
                             onChange={(evt) => setQueryInput(evt.target.value)}
                         />
                         <div className="input-group-append">
-                            <input
-                                className="btn btn-outline-success"
-                                type="submit"
-                                value="Search"
-                            />
+                            <div className="btn-group">
+                                <input
+                                    className="btn btn-outline-success rounded-0"
+                                    type="submit"
+                                    value="Search"
+                                    onClick={(ev) => {
+                                        ev.preventDefault();
+                                        setCurrentQuery({
+                                            query: queryInput,
+                                        });
+                                    }}
```

Comment:
I feel that something is a bit odd in current behavior and toggling counting shouldn't immediately change the query. It should be actually supported by some internal state.

- Initial count state should depend on `count=` and query should be emitted accordingly
- If we toggle count state, it should submit proper `count=` to URI along with submission of the new query e.g. when we click Search on current query or we enter modified/another one